packages/sdk-middleware-http/src/http.js

```diff
@@ -92,7 +92,7 @@ export default function createHttpMiddleware({
     request: MiddlewareRequest,
     response: MiddlewareResponse
   ) => {
-    let abortController
+    let abortController: any
```

Comment:
This is TypeScript and I'm wondering why JavaScript accepts it at all?

Reply:
Cause the node SDK uses flow syntax ;)